students.txt

```diff
@@ -14,7 +14,13 @@ List of students
 7. Daniil I.
 8. Daria P. 
 9. Max R.
-10. Dima Ch.
+10. Dima Chalenko
+Favorite activities
+  .reading books (psychology, proramming)
+  .program
+  .like to sleep a lot
+  .love play gaming
```

Comment:
You can't say that :) You either say `Love play games` or `Love gaming`. Choose one of two and change this line please.